package.json

```diff
@@ -25,7 +25,9 @@
     "bump:lib:version": "cd library && npm --no-git-tag-version --allow-same-version version $VERSION",
     "bump:version": "npm run bump:lib:version",
     "get:name": "cd library && npm run get:name",
-    "get:version": "cd library && npm run get:version"
+    "get:version": "cd library && npm run get:version",
+    "predeploy": "npm run build",
+    "deploy": "cd examples/simple-react && npm run deploy"
```

Comment:
I don't see places where you use `predeploy` or `deploy` scripts, or is it needed by `JamesIves/github-pages-deploy-action@v4`?

Reply:
`predeploy` is automatically run by `deploy` as part of the [NPM lifecycle](https://docs.npmjs.com/cli/v9/using-npm/scripts#pre--post-scripts) 🙂

Reply:
Yes 😄 

I am open to suggestions, but I kept the example and gh-page together for simplicity because it was the easiest to maintain.

Reply:
If it works, I can only accept 😄

Reply:
Well I hope it does, otherwise I just have to provide another PR xD